purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesFactory.kt

```diff
@@ -146,7 +147,7 @@ internal class PurchasesFactory(
     ): DiagnosticsManager {
         return DiagnosticsManager(
             DiagnosticsFileHelper(FileHelper(context)),
-            DiagnosticsAnonymizer(),
+            DiagnosticsAnonymizer(Anonymizer()),
```

Comment:
Nice. Would this run in a background thread? Probably wouldn't want to do a bunch of regex checks in the UI thread.

Reply:
Yup all the code related to diagnostics will run in a background thread. That part will be added in #785 and a future PR to pass in the diagnostics thread.